### ISSUE TITLE
Need some understanding

### ISSUE THREAD

**Issue by swagat123** [opensemanticsearch/open-semantic-search-apps#49]
Hi All 

First of all congratulation for this awesome search engine tool. I was searching for open source semantic search and i came across this tool. I am trying to understand this work flow. I installed this in Debian. I  used few websites through Admin UI  Datasources section to index website  and then set the crawlers. When type a key word for search , it retrieve the website link with key words highlighted. Is it possible to retrieve the Abstract text based on keyword search. Like in science direct if you search based on keywords , it will retrieve the Abstract along with links to pdf/doc files. Is it possible to achieve this using this OSS. If so then how?

Example - http://nactem-copious.man.ac.uk/Thalia/

I appreciate your suggestions.

**Comment by mosea3** [comment]
Hi swagat,

The described feature is on the pending enhancement list.

Currently, there were some issues highlighting PDF direct text (if no OCR result available) and conextual highlighted Website-text.

Stay tuned here!

Best regards

Andy

Von meinem Smartphone gesendet

On 30 Mar 2019, at 11:14, swagat123 <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi All

First of all congratulation for this awesome search engine tool. I was searching for open source semantic search and i came across this tool. I am trying to understand this work flow. I installed this in Debian. I used few websites through Admin UI Datasources section to index website and then set the crawlers. When type a key word for search , it retrieve the website link with key words highlighted. Is it possible to retrieve the Abstract text based on keyword search. Like in science direct if you search based on keywords , it will retrieve the Abstract along with links to pdf/doc files. Is it possible to achieve this using this OSS. If so then how?

Example - http://nactem-copious.man.ac.uk/Thalia/

I appreciate your suggestions.
